src/BondingCurveRegistry.sol

```diff
@@ -21,14 +20,11 @@ import {Errors} from "src/libraries/Errors.sol";
  *         You can think of the registry as a concierge the EthMultiVault uses to access various
  *         economic incentive patterns.
  */
-contract BondingCurveRegistry is Initializable {
```

Comment:
3.8 - Missing _disableInitializers Call in Upgradeable Contracts

We have removed the `is Initializable` property of the contract, and replaced the initializer with a constructor here.